intelmq/bin/intelmqcli.py

```diff
@@ -131,15 +135,19 @@ def init(self):
             self.logger.info("All taxonomies: " + ", ".join(taxonomies))
             for taxonomy in taxonomies:
                 self.logger.info('Handling taxonomy {!r}.'.format(taxonomy))
-                if taxonomy not in lib.SUBJECT or lib.SUBJECT[taxonomy] is None:
+                if (taxonomy not in lib.SUBJECT or lib.SUBJECT[taxonomy] is None) and not self.subject:
```

Comment:
With certat/intelmq#110 merged, lower case values will be enforced for taxonomies.